test/regression/MLH-522_StoryFieldFunctionality_TCT.js

```diff
@@ -0,0 +1,54 @@
+const sel = require('../../data/selectors.json');
+const exp = require('../../data/expected.json');
+
+
+describe ('Story Field functionality', function() {
+
+    describe ('Story Field functionality', function(){
+
+        it ('TCT-004 Story dropdown can choose any story', function () {
+            browser.url('https://qa-apps.netlify.app/app_my_hero');
+            $(sel.storyClick).click();
+            const storyTypes = $$(sel.storyType)[0].isDisplayed();
+            expect(storyTypes).toEqual(true);
+        });
+        it ('TCT-005 Story dropdown can choose only one type of story', function () {
+            browser.url('https://qa-apps.netlify.app/app_my_hero');
+            $(sel.storyClick).click();
+            $$(sel.storyType)[5].click();
+            $(sel.storyClick).click();
+            $$(sel.storyType)[6].click();
+            const showStory = $('.ant-select-selection-item').getAttribute('title');
+            expect(showStory).toEqual(exp.typeOfStory);
+        });
+
+        it ('TCT-006 Story contents', function () {
```

Comment:
Nastya, this test doesn't go through, needs to be fixed

Reply:
i have checked that, all passed through , it is very interesting , first time it gaev me failed the second time passed all

Reply:
<img width="718" alt="Screen Shot 2020-11-17 at 7 04 16 PM" src="https://user-images.githubusercontent.com/70041503/99477706-e6493780-2907-11eb-91a7-7e4219b8a13b.png">